corehq/apps/reports/static/reports/js/filters/case_list_explorer_knockout_bindings.js

```diff
@@ -51,10 +51,17 @@ hqDefine("reports/js/filters/case_list_explorer_knockout_bindings", ['jquery', '
             setTimeout(update, 100);
         },
         update: function(element, valueAccessor){
+            var caseProperties = ko.utils.unwrapObservable(valueAccessor());
             var casePropertyAutocomplete = {
                 getCompletions: function(editor, session, pos, prefix, callback){
-                    var data = ko.utils.unwrapObservable(valueAccessor());
-                    callback(null, _.map(data, function(suggestion){
+                    var currentValue = editor.getValue(),
+                        leftQuotes = (currentValue.substr(0, pos.column).match(/'|"/g) || []).length,
+                        insideQuote =  leftQuotes && (leftQuotes % 2 !== 0);
```

Comment:
Can you have escaped quotes in a case property? That's the only edge case I can think of that is maybe not handled here

Reply:
probably. this also doesn't check if you have something like "hello 'pr33thi'" which I can see being something someone would do. Will update.